src/app/cic/steps.js

```diff
@@ -23,105 +23,258 @@ module.exports = {
+        next: "photoIdExpiry",
+      },
+      "nameEntry",
+    ],
   },
   "/brpDetails": {
     fields: ["brpExpiryDate"],
     controller: brpDetails,
     editable: true,
     editBackStep: "checkDetails",
-    next: brpDetails.prototype.next
+    next: [
+      {
+        field: "brpExpiryDate",
+        op: "before",
+        value: "today",
+        next: "photoIdExpiry",
+      },
+      {
+        field: "brpExpiryDate",
+        op: "after",
+        value: "2025-01-01",
+        next: "photoIdExpiry",
+      },
+      "nameEntry",
+    ],
   },
   "/euIdentityCardDetails": {
     fields: ["euIdCardExpiryDate"],
     controller: euIdentityCardDetails,
     editable: true,
     editBackStep: "checkDetails",
-    next: euIdentityCardDetails.prototype.next
+    next: [
+      {
+        field: "euIdCardExpiryDate",
+        op: "before",
+        value: "today",
+        next: "photoIdExpiry",
+      },
+      {
+        field: "euIdCardExpiryDate",
+        op: "after",
+        value: "75 years",
+        next: "photoIdExpiry",
+      },
+      "nameEntry",
+    ],
   },
   "/youngScotNecDetails": {
     fields: ["youngScotNationalEntitlementCardExpiryDate"],
     controller: youngScotNationalEntitlementCardDetails,
     editable: true,
     editBackStep: "checkDetails",
-    next: youngScotNationalEntitlementCardDetails.prototype.next
+    next: [
+      {
+        field: "youngScotNationalEntitlementCardExpiryDate",
+        op: "before",
+        value: "today",
+        next: "photoIdExpiry",
+      },
+      {
+        field: "youngScotNationalEntitlementCardExpiryDate",
+        op: "after",
+        value: "15 years",
+        next: "photoIdExpiry",
+      },
+      "nameEntry",
+    ],
   },
   "/citizenCardDetails": {
     fields: ["citizenCardExpiryDate"],
     controller: citizenCardDetails,
     editable: true,
     editBackStep: "checkDetails",
-    next: citizenCardDetails.prototype.next
+    next: [
+      {
+        field: "citizenCardExpiryDate",
+        op: "before",
+        value: "today",
+        next: "photoIdExpiry",
+      },
+      {
+        field: "citizenCardExpiryDate",
+        op: "after",
+        value: "4 years",
+        next: "photoIdExpiry",
+      },
+      "nameEntry",
+    ],
   },
   "/euPhotocardDlDetails": {
     fields: ["euPhotocardDlExpiryDate"],
     controller: euPhotocardDlDetails,
     editable: true,
     editBackStep: "checkDetails",
-    next: euPhotocardDlDetails.prototype.next
+    next: [
+      {
+        field: "nonUKPassportExpiryDate",
+        op: "before",
+        value: "today",
+        next: "photoIdExpiry",
+      },
+      {
+        field: "euPhotocardDlExpiryDate",
+        op: "after",
+        value: "75 years",
+        next: "photoIdExpiry",
+      },
+      "nameEntry",
+    ],
   },
-  '/photoIdExpiry': {
-    next: "/photoIdSelection"
+  "/photoIdExpiry": {
+    editable: true,
+    editBackStep: "checkDetails",
+     next: "/photoIdSelection",
```

Comment:
nit: do we need "/"?

Reply:
Removed `editable` from "/photoIdExpiry" - does not seem to make a difference